src/Driver/SqsDriver.php

```diff
@@ -171,19 +172,7 @@ protected function resolveUrl($queueName)
             return $this->queueUrls[$queueName];
         }
 
-        try {
-            $result = $this->sqs->getQueueUrl(['QueueName' => $queueName]);
-        } catch (SqsException $exception) {
-            if ($exception->getExceptionCode() === 'AWS.SimpleQueueService.NonExistentQueue') {
-                throw new SqsException(
-                    'The queue "' . $queueName . '" is neither aliased locally to an SQS URL nor could it be resolved by SQS.',
-                    $exception->getCode(),
-                    $exception
-                );
-            }
-
-            throw $exception;
-        }
```

Comment:
`Aws\Sqs\Exception\SqsException` is not imported in this class, so this block would only catch `Bernard\Driver\SqsException`, which [does not exist](https://scrutinizer-ci.com/g/bernardphp/bernard/inspections/e8374af3-5545-4ac7-a56d-aae3ccb094d4/issues/files/src/Driver/SqsDriver.php?status=fixed&orderField=path&order=asc&honorSelectedPaths=0).

The driver could be updated to include a proper import of the exception, but the constructor signature is different from v2 to v3 of the SDK, so this would be a blocker to support both versions. Additionally, in v2 there are several subclasses of `SqsException` that could be thrown by an SqsClient, and this catch block squashes them all into the base `SqsException` class, potentially losing some valuable type information.